src/s2i/bin/assemble

```diff
@@ -72,10 +72,20 @@ fix-permissions /opt/app-root -P
 # * pip < 9 does not support different packages' versions for Python 2/3
 # * pip < 19.3 does not support manylinux2014 wheels. Only manylinux2014 (and later) wheels
 #   support platforms like ppc64le, aarch64 or armv7
+#
+# Also pip 9 we have in RHEL 8 can use metadata about supported Python version
+# so we have to limit the version of pip only for RHEL 7 where the default pip
+# version is 7. And because pip still supports Python 3.8, the limit makes sense
+# only for Python 2.7.
```

Comment:
I'm not sure I understand the latest sentence. I thought this makes sense only for Python 2.7 in RHEL 7 is because there the pip version is too old to use the metadata. What does it have to do with pip supporting Python 3.8?